PyVistaInterface is using `off_screen` without checking if it is part of `plotter_kwargs`

I believe we should check that argument is not passed already.


```pytb
mapdl = <ansys.mapdl.core.mapdl_grpc.MapdlGrpc object at 0x7fb92a0c5c10>
make_block = None

    @requires("pyvista")
    def test_plotter_input(mapdl, make_block):
        import pyvista as pv
    
>       pl = MapdlPlotter(off_screen=False)

tests/test_plotting.py:879: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/__t/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/plotting/visualizer.py:117: in __init__
    self._backend = MapdlPlotterBackend(use_trame=use_trame, **plotter_kwargs)
/__t/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/mapdl/core/plotting/visualizer.py:71: in __init__
    super().__init__(
/__t/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/tools/visualization_interface/backends/pyvista/pyvista.py:156: in __init__
    self._pl = PyVistaInterface(show_plane=show_plane, use_qt=use_qt, **plotter_kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ansys.tools.visualization_interface.backends.pyvista.pyvista_interface.PyVistaInterface object at 0x7fb8f01c1210>
scene = None, color_opts = None, num_points = 100, enable_widgets = True
show_plane = False, use_qt = False, plotter_kwargs = {'off_screen': False}

    def __init__(
        self,
        scene: Optional[pv.Plotter] = None,
        color_opts: Optional[Dict] = None,
        num_points: int = 100,
        enable_widgets: bool = True,
        show_plane: bool = False,
        use_qt: bool = False,
        **plotter_kwargs,
    ) -> None:
        """Initialize the plotter."""
        # Generate custom scene if ``None`` is provided
        if scene is None:
            if viz_interface.TESTING_MODE:
                if use_qt and _HAS_PYVISTAQT:
                    scene = pyvistaqt.BackgroundPlotter(off_screen=True)
                else:
                    if use_qt and not _HAS_PYVISTAQT:
                        message = "PyVistaQt dependency is not installed. Install it with " + \
                                  "`pip install ansys-tools-visualization-interface[pyvistaqt]`."
                        logger.warning(message)
>                   scene = pv.Plotter(off_screen=True, **plotter_kwargs)
E                   TypeError: pyvista.plotting.plotter.Plotter() got multiple values for keyword argument 'off_screen'

/__t/Python/3.11.10/x64/lib/python3.11/site-packages/ansys/tools/visualization_interface/backends/pyvista/pyvista_interface.py:91: TypeError
```

From: https://github.com/ansys/pymapdl/actions/runs/12274602876/job/34248062339?pr=3612#step:11:3745

--- a/doc/changelog.d/207.fixed.md
+++ b/doc/changelog.d/207.fixed.md
@@ -1,0 +1,1 @@
+fix: avoid passing twice same argument

--- a/src/ansys/tools/visualization_interface/backends/pyvista/pyvista_interface.py
+++ b/src/ansys/tools/visualization_interface/backends/pyvista/pyvista_interface.py
@@ -88,6 +88,8 @@ class PyVistaInterface:
                         message = "PyVistaQt dependency is not installed. Install it with " + \
                                   "`pip install ansys-tools-visualization-interface[pyvistaqt]`."
                         logger.warning(message)
+                    # Avoiding having duplicated argument
+                    plotter_kwargs.pop("off_screen", None)
                     scene = pv.Plotter(off_screen=True, **plotter_kwargs)
             elif use_qt:
                 scene = pyvistaqt.BackgroundPlotter()

